apps/zidence-api/src/app/middlewares/properties/controller.ts

```diff
@@ -93,10 +95,50 @@ export const getProperties = async (req: Request, res: Response) => {
       properties,
     })
   } catch (error) {
-    console.error(error.message)
     res.status(400).json({
       message: 'Get all properties failed',
       error: error.message,
     })
+    console.error(error.message)
   }
 }
+
+
+
+export const getProperty = async (req: Request, res: Response) => {
+  try {
+    const { id } = req.params
+
+    const property = await prisma.properties.findUnique({
```

Comment:
I think the model has changed into singular form `property` and has been merged to main. So use `prisma.property` instead of `prisma.properties`